### PR TITLE
Update hifiasm v.0.16.1

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR includes the following modifications:
- --max-kocc parameter added
- --hg-size parameter added
- --hom-cov parameter added

### CODE REVIEW

**Comment by gallardoalba** [comment]
> mh, hifiasm-out7-processed.gfa  are empty is that ok?

I need to check it.

**Comment by gallardoalba** [comment]
> mh, hifiasm-out7-processed.gfa are empty is that ok?

It requires an integer value but followed by a character (k,m,g)? I tried it by providing a string, but doesn't work.

Screenshot from 2021-09-22 10-47-54

**Comment by gallardoalba** [comment]
Fixed @bgruening

File: tools/hifiasm/hifiasm.xml [hunk 1]
```diff
@@ -1,14 +1,15 @@
-<tool id="hifiasm" name="Hifiasm" version="@VERSION@+galaxy0">
+<tool id="hifiasm" name="Hifiasm" version="@TOOL_VERSION@+galaxy@SUFFIX_VERSION@">
     <description>haplotype-resolved de novo assembler for PacBio Hifi reads</description>
     <macros>
-        <token name="@VERSION@">0.15.4</token>
+        <token name="@TOOL_VERSION@">0.16.1</token>
+        <token name="@SUFFIX_VERSION@">0</token>
```
Comment (simonbray) [inline tools/hifiasm/hifiasm.xml:5]:
```suggestion
        <token name="@VERSION_SUFFIX@">0</token>
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**